internal/node_client/rpc_client_test.go

```
@@ -0,0 +1,37 @@
+package node_client_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/bitcoin-sv/arc/internal/node_client"
+)
+
+func TestRPCClient(t *testing.T) {
+	t.Helper()
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+
+	ctx := context.Background()
```

Want's the main idea to check if ctx worked? so cancelling it should return the corresponding error?

---

Sorry, don't get the question.
Are you suggesting to test cancelling the context? That would make sense